server/index.js

```diff
@@ -39,6 +40,22 @@ app.post('/user/add', (req, res) => {
 	);
 });
 
+// Store loginIds so that we can use it to verify logins later
+app.post('/user/login', (req, res) => {
```

Comment:
i was thinking to use mongodb id for this. Basically when a user creates an account MongoDB gives a unique id, going to do that

Reply:
Oh Ok, I get where you're coming from....

The main reason for creating this route was to store *new unique* ids that we can authenticate later (in case a user forges it) for each new login.

But I think your point might cause any of these issues:
* We still go back to first issue where by any user can login using any email. By this, i mean if a user knows the mongodb id of another user (from users document) which is constant, they can easily login since the id won't be changed after first login (based on the current code in Login.jsx where users are only registered when their emails don't exist in the db)
* This won't work for anonymous users since we're not storing their email after login. Even if we do so using a unique id, we still fall back to the above issue.

Reply:
> Alright, Why would it be needed in backend if they don't want to login in. Everything about them will be stored in local storage.
> 
> They are not the main priority now

To prevent users from forging their own loginIds... Essentially, when the provided Id does not exist in the db, then it's fake/forged by the user. *Without extra added features that would be authenticated, any user can view other users' chats (email logins) just by knowing their emails*

Reply:
Perhaps I can remove that for now then we open a new issue to improve on security of the system later?